src/chompfile.rs

```diff
@@ -215,13 +208,9 @@ impl ChompTaskMaybeTemplated {
     pub fn targets_vec(&self) -> Result<Vec<String>> {
         if let Some(ref target) = self.target {
             let target_str = target.to_string();
-            validate_path(&target_str)?;
             Ok(vec![target_str])
         } else if let Some(ref targets) = self.targets {
             let targets = targets.clone();
-            for target in &targets {
-                validate_path(&target)?;
```

Comment:
What if we literally replaced `validate_path` in these loops with doing the canonicalization here at this lower level rather than at all consumer positions?